client/app/components/labs-ui/site-header.js

```diff
@@ -0,0 +1,14 @@
+import Component from '@ember/component';
+
+export default Component.extend({
```

Comment:
Is the new new site-header, site-title component because you removed labs-ui?

Reply:
@godfreyyeung  yup it's the copied over version. see updated description for why